### DISCUSSION TITLE
24.9.19版本的分流突然出问题了

### DISCUSSION THREAD

**Discussion by sofs2005** [XTLS/Xray-core#3846] [Q&A]
应该是昨天半夜或者今天早上突然就挂了，在pw或者pw2中开启分流要么就是会自动退出要么就是cpu跑满，不用分流功能就没问题，使用的同一个节点，配置也没改过，有人知道会是什么问题吗

**Comment by XavierPENROSE** [comment]
我也遇到了相似的问题，我现在退回到1.8.24了。

**Comment by RPRX** [comment]
没用过，不了解这个“分流”对应的是 Xray 的哪个功能，如果是 Xray 的问题且多人复现，请发个 issue

**Comment by Pacalini** [comment]
24.9.30 是否还有问题？日志呢？pw/pw2 发版慢，有条件拉最新源码编译试试
